MacrobenchmarkSample/baseBenchmarks/src/main/java/com/example/benchmark/macro/base/scroll/ScrollBenchmark.kt

```diff
@@ -38,6 +39,8 @@ import org.junit.runner.RunWith
  * Scrolls down and up in the default activity.
  */
 @RunWith(AndroidJUnit4::class)
+// These tests are only useful for apps that have a scrollable view in the benchmarked activity.
+@Suppress
```

Comment:
Could we leverage JUnit `assumeTrue` for that?

Like:
- getting the UiDevice within the test (but before the `rule.measureRepeated`)
- checking whether `assumeTrue(device.hasObject(By.scrollable()))` (pseudocode)

Reply:
We could find another way but that's currently not a priority. Filed #207 to keep track of this.